### DISCUSSION TITLE
Digital recognition badges for CNCF project maintainers

### DISCUSSION THREAD

**Discussion by aliok** [cncf/toc#1248] [Ideas]
Recognition of contributions via digital badges is a great incentive for people to contribute.

In open source and more specifically at CNCF, the most valuable resources are contributors. Many projects are struggling with growing their contributor base and we need more incentives for people to stick around and become maintainers.

I know CNCF (or LF) is giving Credly badges to people like KubeCon speakers. If LF/CNCF has an agreement with Credly that CNCF projects can leverage, it would be great.

Related tasks where community members asked for such badges:
- https://github.com/knative/community/issues/1493
- https://github.com/cncf/mentoring/issues/1105
- https://github.com/cncf/toc/issues/1127

Not sure if this is a TOC responsibility or a https://github.com/cncf/foundation one.

**Comment by aliok** [comment]
The digital badges won't solve everything, but it might be a good incentive.

My gut feeling is that people that get those badges add them to their profile. This is a good promotion for CNCF and it would be a good promotion for projects.

To make things more concrete though, it **might** be possible to make some measurements to have a better understanding of a possible effect.

- AFAIK, when you're given a badge, you go and open the Credly system to "accept" it (or add it to your profile, or something like that).  Perhaps it is possible to get the numbers for given badges and their acceptance rate.
- Similarly, people often post these badges on LinkedIn or put them in their profile as a "certification". I am not sure about the limitations of LinkedIn API, but I would be super interested in to see what kind of percentage is putting these badges on their profile.

My point is, if possible, try to understand the utilization of the given badges first.

**Comment by linsun** [comment]
This is a very interesting idea - if we do make these digital badges, would be great to cover it for the year while the maintainer is active.

**Comment by amye** [comment]
We paused on implementing Credly because of the challenge it posed to maintaining it, I realized it would quickly devolve into a spreadsheet of doom - I think there are now more varied ways and possibly easier to handle it! 
Can we move this into https://github.com/cncf/toc/issues/1127 and revisit the conversation there?

**Reply by aliok** [reply]
I wasn't only thinking about TAGs (#1127 is about TAGs), but it is OK to revisit this matter after getting some output in #1127 and selecting what's generally applicable to CNCF projects as well.

**Comment by TheFoxAtWork** [comment]
Quick update here!  heard that there is progress being made here. not quite ready yet but its moving forward!

**Reply by aliok** [reply]
Looking forward to seeing the progress here! It would be such a nice recognition mechanism.

**Comment by dims** [comment]
+1 to this! Hopefully we recognize folks for specific preceding year or something so there is motivation to continue collecting and those who are viewing the badges know what timeframe the contributions were for. (and that they are not open ended).

**Comment by charliemah** [comment]
There is an open source alternative to Credly for issuing digital recognition badges called Tahrir, which is used by the Fedora Project to help both new and current contributors in exploring various ways to engage.
https://github.com/fedora-infra/tahrir
https://badges.fedoraproject.org

**Comment by charliemah** [comment]
The Fedora project is currently developing Fedora Badges 2.0. More details can be found at: https://youtu.be/SRvf8v7Zurw (An update on the progress for the ongoing revamp of the whole Fedora Badges system).
Another great example of promoting OSS community sustainability through digital badges is the CHAOSS DEI Badging Initiative (https://github.com/badging/badging MIT license). This initiative awards digital badges to OSS projects and events, highlighting their commitment to enhancing diversity, equity, and inclusion.
